README.md

```diff
@@ -140,6 +140,22 @@ func application(_ application: UIApplication, didFailToRegisterForRemoteNotific
 }
 ```
 
+### Location pushes
+
+Starting with iOS 15, Apple supports power efficient way to request location trough [location push service extension](https://developer.apple.com/documentation/corelocation/creating_a_location_push_service_extension).
+
+To use this new functionality you should obtain Apple entitlments, create location push service extension and make some tweaks in your push notifications code within your app:
```

Comment:
I don't think we should be telling the user how to configure their app to receive location push notifications. Apple’s documentation already does that better than we're going to, plus we have an example app. We should just be describing how to use our SDK.

Reply:
To be clear — I’m saying that we don't need to tell the user how to configure _their app_ to use location push notifications. We do however need to document how to configure _our example app_ to use location push notifications.

Reply:
https://github.com/ably/ably-cocoa/pull/1771/commits/794e634e187b3df8f0a56ffac1243e41e592211e